cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel_scalar.hpp

```diff
@@ -87,6 +90,8 @@ vertex_similarity_result call_jaccard_default_kernel_scalar(
                                    float(i_neighbor_size + j_neighbor_size - intersection_value);
                     first_vertices[nnz] = i;
                     second_vertices[nnz] = j;
+                    // Safe incrementing of nnz
+                    //max nnz = (2^(31)  * 2^(31))=(2^62) < 2^63 = max size of std::int64_t
```

Comment:
Do you want to leave comment here, because it is a justification of overflow checking absence?

Reply:
Yes.